CalibTracker/SiPixelESProducers/plugins/alpaka/SiPixelCablingSoAESProducer.cc

```diff
@@ -0,0 +1,157 @@
+#include "CondFormats/DataRecord/interface/SiPixelFedCablingMapRcd.h"
+#include "CondFormats/DataRecord/interface/SiPixelQualityRcd.h"
+#include "CondFormats/SiPixelObjects/interface/SiPixelFedCablingMap.h"
+#include "CondFormats/SiPixelObjects/interface/SiPixelFedCablingTree.h"
+#include "CondFormats/SiPixelObjects/interface/SiPixelQuality.h"
+#include "DataFormats/SiPixelClusterSoA/interface/ClusteringConstants.h"
+#include "CondFormats/SiPixelObjects/interface/SiPixelMappingHost.h"
+#include "CondFormats/SiPixelObjects/interface/alpaka/SiPixelMappingDevice.h"
+#include "FWCore/Framework/interface/ESTransientHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/ESGetToken.h"
+
+#include "Geometry/CommonDetUnit/interface/GeomDetType.h"
+#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
+#include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
+
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/ESProducer.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/ModuleFactory.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+
+#include "RecoTracker/Record/interface/CkfComponentsRecord.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  using namespace cms::alpakatools;
+
+  class SiPixelCablingSoAESProducer : public ESProducer {
+  public:
+    SiPixelCablingSoAESProducer(edm::ParameterSet const& iConfig)
+        : ESProducer(iConfig), useQuality_(iConfig.getParameter<bool>("UseQualityInfo")) {
+      auto const& component = iConfig.getParameter<std::string>("ComponentName");
```

Comment:
I repeat my earlier comment that in principle this kind of `ComponentName` is redundant, as setting the data label could be achieved with the framework's built-in `appendToDataLabel` parameter that is available for all ESProducers.